### DISCUSSION TITLE
AB Scripting - Display Format

### DISCUSSION THREAD

**Discussion by longj08** [sadieblue/agilityblue.workspace-scripts#22] [Q&A] [answered]
For a reference field linked to the Media Log, can the “DisplayFormat” be set to any field within the Media Log specifically any custom field added to it?  

Currently it is set up as:

•	DisplayFormat = "#%Ref% - %Val%"

Which display the selected media log entry ID and Type.

Any help is appreciated. Thank you!

**Comment by bmward80** [comment, marked as answer]
Hi @longj08,

No, the tokenized fields cannot represent any other fields. It’s not recommended to alter the display values of reference fields because the system can update the cache at any time across the database for all references when it senses changes to the underlying object and your custom changes would end up reverting back. 

Brandon
